src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag

```diff
@@ -78,6 +80,35 @@
           </div>
         </div>
         <br class="hidden-xs">
+        <div class="row">
+          <div class="col-sm-12">
+            <div class="form-group">
+              <h5 class="col-sm-2">
+                <label class="control-label" for="<%= Const.ParamsNames.COURSE_NAME %>">
+                  Course name
+                </label>
+              </h5>
+              <div class="col-sm-10 col-md-8">
+                <c:choose>
+                  <c:when test="${fsForm.courseIdEditable}">
+                    <c:forEach items="${courseAttributes}" var="attributes">
+                      <input class="course-name-data" type="hidden"
+                             value="[${attributes.id}][${attributes.name}]">
+                    </c:forEach>
+                    <h5 id="<%= Const.ParamsNames.COURSE_NAME %>" class="form-control-static font-weight-normal">
+                    </h5>
```

Comment:
What is this empty `<h5>` block for?

Reply:
In my Javascript, this h5 tag is for me to insert the name of the course name